monitoring/instrumented_mutex.h

```diff
@@ -49,6 +68,7 @@ class InstrumentedMutex {
   Statistics* stats_;
   SystemClock* clock_;
   int stats_code_;
+  InstrumentedCondVar* bg_cv_;
```

Comment:
This looks like it pushes `sizeof(InstrumentedMutex)` over the cache line size (64 bytes), though I didn't verify. In any case it might be better to conditionally include this field only when `COERCE_CONTEXT_SWITCH` is set.

Reply:
Fixed